keyboards/bioi/morgan65/rules.mk

```diff
@@ -16,7 +16,5 @@ AUDIO_ENABLE = no           # Audio output
 LTO_ENABLE = yes            # Reduce firmware size
 BLUETOOTH_ENABLE = yes
 
-VIA_ENABLE = yes            # VIA support should be enabled here due to the main() loop will be compiled first.
```

Comment:
Good catch, VIA should not be enabled at the keyboard level.

Reply:
> Good catch, VIA should not be enabled at the keyboard level.

indeed, and the comment about it did not make any sense either. I should probably have moved the keyboard specifics in a new PR, but here they are. Any remaining merge blockers I should consider?